### PR TITLE
Refactor Portfolio Model Tests Ready For Review

### PR DESCRIPTION
**Pull request by isabeckman**
In PortfolioModelTest.java, there are two new tests for review. 

Test 1: portfolioSectionTest()
To loosely couple the Portfolio model, we are going to get rid of all the fields including About Me, Education, Industry Equivalency, Projects, and Skill Matrix. These will be replaced with a List of ArrayLists of type Interface called Portfolio Section. This is meant to create a separation of concerns. Each section (or class such as Education, Projects, etc.) is a sub-ArrayList. There will be one ArrayList per section of the portfolio. This gives the options to add, remove or edit sections of the portfolio.

Test 2: buildLooselyCoupledPortfolio() 
Tests the build of a portfolio with new field.

### CODE REVIEW

**Review by foshaystephanie — CHANGES_REQUESTED** [review]
It looks good, but your .metadata/.plugins/ files are coming through in the merge. That file looks like it's concerned with your Eclipse UI. It could contribute to conflicts later down the road. Add it to your .gitignore (following the recommendations in the stackoverflow link) and resubmit the changes. Don't include the .metadata root folder in your .gitignore, just the /.plugins.

https://stackoverflow.com/questions/4278601/what-of-eclipse-project-metadata-can-i-safely-ignore-in-git-mercurial

File: Portfolio-Review-Service/src/test/java/PortfolioReviewService/models/PortfolioModelTest.java [hunk 1]
```diff
@@ -1,53 +1,113 @@
...
-		p.setAboutMe(null);
-		p.setEducation(null);
-		p.setIndustryEquivalency(null);
-		p.setMyUser(null);
-		p.setProjects(null);
-		p.setSkillMatrix(null);
-		
-		return p;
-	}
+//	private Portfolio buildPortfolio() {
```
Comment (KennethDavis391) [inline Portfolio-Review-Service/src/test/java/PortfolioReviewService/models/PortfolioModelTest.java]: If these tests are no longer needed, please remove them completely.

**Review by KennethDavis391 — CHANGES_REQUESTED** [review]

**Review by foshaystephanie — CHANGES_REQUESTED** [review]
Take out commented code, and add "bin" to .gitignore

**Comment by foshaystephanie** [comment]
For consistency, don't delete anything from the Portfolio-Review-Service .gitignore. Add those contents back in. I'm also still getting bin files in this merge. Ideally, nothing except files you've manually edited should get merged.

**Review by foshaystephanie — APPROVED** [review]

**Review by KennethDavis391 — APPROVED** [review]

**Pull request merged**